### PR TITLE
adds strategic plan blog post

### PR DESCRIPTION
**Pull request by katyhuff**
This post should ideally go up sometime tomorrow. Open to edits until then.

### CODE REVIEW

**Comment by jduckles** [comment]
@katyhuff looks great, thanks for putting it together. I'll wait a bit for any other comments and merge at mid-day.

**Comment by gvwilson** [comment]
May I please have until end of day Pacific time to review this?

**Comment by jduckles** [comment]
Sure.

File: _posts/2016/01/2016-01-11-strategic-plan.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+title: "A Strategic Plan for the Software Carpentry Foundation"
+date: 2016-01-11
+time: "03:00:00"
+category: ["Software Carpentry Foundation"]
+---
+
+The Software Carpentry Foundation Steering Committee held a strategic planning
+meeting in August. In that two-day, intensive meeting, we focused on developing
+a long term strategic plan for the Software Carpentry Foundation.  The process
+focused on identification of our stakeholders and mission. It also used the
```
Comment (gvwilson) [inline _posts/2016/01/2016-01-11-strategic-plan.md]: double use of "focused"

File: _posts/2016/01/2016-01-11-strategic-plan.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+already moved forward since the development of the plan (such as the new
+website design, which contributes to reporting and transparency).
+
+- Reporting and Transparency
+- Documentation of Procedures
+- Instructor Pipeline Management
+- Coordination with Data Carpentry
+
+More information about the detailed meaning of these issues can be found [in
+the docpument
```
Comment (gvwilson) [inline _posts/2016/01/2016-01-11-strategic-plan.md]: "document" not "docpument"

File: _posts/2016/01/2016-01-11-strategic-plan.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+itself](https://github.com/swcarpentry/board/blob/master/strategy/strategic_plan.md).
+
+Rather than a top-down declaration of our goals as a foundation, we intended
+this document to capture what we understood to be the ideas and mission of you,
```
Comment (gvwilson) [inline _posts/2016/01/2016-01-11-strategic-plan.md]: "We believe this plan captures the ideas and missions of _you_, our community."

File: _posts/2016/01/2016-01-11-strategic-plan.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+our community. As you look over this document, we hope you won’t hesitate to
+provide feedback that will help SCF clarify its mission, nurture its strengths,
```
Comment (gvwilson) [inline _posts/2016/01/2016-01-11-strategic-plan.md:48]: How do you want feedback? Comments on this post?

File: _posts/2016/01/2016-01-11-strategic-plan.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+and reach its aspirations. If you have any questions or suggestions, we hope
+you'll get in touch.
+
+Even better, if you’d like to be instrumental in the annual revision of this
+strategic plan, we hope you might consider [running for the 2016 Steering
```
Comment (gvwilson) [inline _posts/2016/01/2016-01-11-strategic-plan.md]: s/we hope you might consider/please consider/

File: _posts/2016/01/2016-01-11-strategic-plan.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+Committee.](http://software-carpentry.org/blog/2015/12/call-for-candidates-elections-2016.html)
```
Comment (gvwilson) [inline _posts/2016/01/2016-01-11-strategic-plan.md]: The deadline is this Friday.

**Comment by gvwilson** [comment]
Only minor comments - thanks for giving me a chance to look it over.

**Comment by katyhuff** [comment]
ok! I made the changes. Thanks for looking it over @gvwilson !  I'm going to change the name of the file so that it appears today. Ok to pull it this afternoon @jduckles ?

**Comment by jduckles** [comment]
Pulling now.

**Comment by katyhuff** [comment]
ack! I hadn't quite changed the date/filename before you pulled!

**Pull request merged**